ludwig/data/dataset/ray.py

```diff
@@ -64,14 +67,20 @@ def read_remote_parquet(path: str):
 
 
 class RayDataset(Dataset):
-    """Wrapper around ray.data.Dataset."""
+    """Wrapper around ray.data.Dataset.
+
+    Attributes:
+        auto_window: If True and the dataset is larger than available memory,
+            automatically set window size to `<available memory> // 5`.
+    """
 
     def __init__(
         self,
         df: Union[str, DataFrame],
         features: Dict[str, Dict],
         training_set_metadata: Dict[str, Any],
         backend: Backend,
+        auto_window: bool = False,
```

Comment:
If we want to configure this, should we propagate this parameter here: https://github.com/ludwig-ai/ludwig/blob/master/ludwig/data/dataset/ray.py#L157? 

This is where we initialize the RayDataset object after preprocessing.

Reply:
Added the passthrough in [24f36f5](https://github.com/ludwig-ai/ludwig/pull/2721/commits/24f36f5829d2197a39ff847dc016fd2108341052) and updated the tests to use it in [1c71308](https://github.com/ludwig-ai/ludwig/pull/2721/commits/1c71308fc403044402d53ab9e9d39da94ef71794).